wod_predictor/data_loader.py

```diff
@@ -99,9 +99,22 @@ def load_athlete_info(self):
         athlete_info = {}
         for file in files:
             if file.endswith("info.csv"):
-                year = file.split("_")[0]
+
+                # Parse the file name for year and gender
+                file_name_split = file.split("_")
+                year = file_name_split[0]
+                gender = file_name_split[1]
+
+                # Create the DataFrame from the file content
                 df = pd.read_csv(os.path.join(self.root_path, file))
 
+                if gender == 'Mens':
+                    df['gender'] = 0
```

Comment:
Could it be named to something more informative? `gender_is_female` would be more descriptive for other teams, otherwise it might be confusing to know which value indicates 1.

Reply:
I can leave that as male and female. But that is the only column with data type as string in the dataset. We can use encoder later in the model pipeline.

Reply:
Oh I meant the column name can be made more informative. You're right about the value set to binary, its a good call we should keep it numeric.